DataFormats/CTPPSReco/src/classes_def.xml

```diff
@@ -90,6 +90,23 @@
   <class name="edm::DetSetVector<CTPPSPixelRecHit>"/>
   <class name="edm::Wrapper<edm::DetSetVector<CTPPSPixelRecHit> >"/>
 
+  <class name="CTPPSPixelLocalTrack" ClassVersion="3">
+    <version ClassVersion="3" checksum="166602192"/> 
+  </class>
+  <class name="edm::DetSet<CTPPSPixelLocalTrack>"/>
+  <class name="std::vector<CTPPSPixelLocalTrack>"/>
+  <class name="std::vector<edm::DetSet<CTPPSPixelLocalTrack> >"/>
+  <class name="edm::DetSetVector<CTPPSPixelLocalTrack>"/>
+  <class name="edm::Wrapper<edm::DetSetVector<CTPPSPixelLocalTrack> >"/>
+  <class name="edm::DetSetVector<CTPPSPixelLocalTrack::CTPPSPixelFittedRecHit>"/>
+  <class name="edm::Wrapper<edm::DetSetVector<CTPPSPixelLocalTrack::CTPPSPixelFittedRecHit>>"/>
+  <class name="std::vector<edm::DetSet<CTPPSPixelLocalTrack::CTPPSPixelFittedRecHit> >"/>
+  <class name="std::vector<CTPPSPixelLocalTrack::CTPPSPixelFittedRecHit>"/>
+  <class name="math::GlobalPoint"/>
+  <class name="edm::Wrapper<math::GlobalPoint>"/>
+  <class name="CTPPSPixelFittedRecHit"/>
+  <class name="edm::Wrapper<CTPPSPixelFittedRecHit>"/>
```

Comment:
Wrapper dictionaries are typically not needed for types that are saved only in containers.
Please check and drop.